pkg/registry/registry.go

```diff
@@ -25,14 +25,25 @@ func RegisterServiceWithPriority(instance Service, priority Priority) {
 }
 
 func RegisterService(instance Service) {
-	services = append(services, &Descriptor{
+	Register(&Descriptor{
```

Comment:
Just wanted to confirm you meant to exclude the `RegisterServiceWithPriority` func from using `Register`?

Reply:
@wbrowne I will check.

Reply:
@wbrowne Looking at the code, I simply didn't think of it

Reply:
Thanks, made the change!